about.html

```diff
@@ -0,0 +1,121 @@
+        </p>
+      </div>
+      <p class="contact-about">Please contact us per Email for any further questions about ETHIO WEBSITE DEVELOPMENT COURSES!</p>
+      <p class="email-about">ethi-web@ethioweb.com</p>
+    </div>
+  </header>
+
+  <section class="about-section">
+    <h3 class="abt-sub-title">Creative Commons</h3>
+    <h2 class="abt-title">Ethio Website Development</h2>
+    <hr class="underline"/>
+    <p>
+      Launch is the fourth stage of website development and is responsible for publicizing and promoting the website. This includes setting up an online presence including a domain name and web hosting, creating marketing materials, and advertising the website.
+    </p>
+    <div class="about-image">
+      <img src="./images/logo-no-background.png" alt="about logo image" class="my-image">
+    </div>
+    <hr/>
+  </section>
+  <section class="about-gallery">
+    <h2 class="gallery-title">See the past Graduates of Ethio Web</h2>
+    <hr class="underline"/>
+    <p>Take a look at the last two gradutes students which took place in Addis Ababa and England, Europe.</p>
+    <div class="my-gallery">
+      <div class="image image-one">
+        <h2>2021</h2>
+        <h3>Ethio Web Graduates of 2021, Addis Ababa</h3>
+      </div>
+      <div class="image image-two">
+        <h2>2022</h2>
+        <h3>Ethio Web Graduates of 2022, England</h3>
+      </div>
+    </div>
+  </section>
+
+  <footer>
+    <div class="partner-wrap">
+      <h2 class="partner">Partner</h2>
+      <hr class="underline under-center" />
+      <div class="partner-imgs">
+          <img src="./images/mozilla2.png" alt="partner1" class="partner1">
+          <img src="./images/amazon.png" alt="partner2" class="partner2">
+          <img src="./images/netflix.png" alt="partner3" class="partner3">
+          <img src="./images/mozilla2.png" alt="partner4" class="partner4">
+          <img src="./images/airbnb2.png" alt="partner5" class="partner5">
+      </div>
+    </div>
+    <div class="foot">
+      <div class="footer-logo">
+        <img src="./images/logo-no-background.png" alt="foot log"/>
+      </div>
+      <div>
+        <p class="foot-title">2023 Ethio web development. Some Right Reserved</p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- Kindly make sure to match the given design, which includes:
   - [ ] Removing the "Partners" section on the "About" page.
   - [ ] Changing the background color of the footer section to match the given design on the "About" page.
   - [ ] Adjusting the spacing between the two footer elements on mobile view on the "Home" and "About" pages.
   - [ ] Removing the footer section in the "Home" page in mobile view to match the given design.
   
| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/218228510-c6d2bd44-48cd-4278-b1bf-915023f2be8d.png) | ![image](https://user-images.githubusercontent.com/62290545/218231590-8bcd59c1-5e3a-4086-a660-9947e2914a28.png) |